packages/components/inputs/search-select-input/src/search-select-input.js

```diff
@@ -0,0 +1,213 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import {
+  SafeHTMLElement,
+  filterAriaAttributes,
+} from '@commercetools-uikit/utils';
+import SearchIconDropdownIndicator from '../../../../../src/components/internals/search-icon-dropdown-indicator';
+import AsyncSelectInput from '../../async-select-input';
+import { SearchSelectInputOption } from './search-select-input-option';
+import messages from './messages';
+
+const SearchSelectInput = (props) => {
+  const intl = useIntl();
+  const noOptionsMessage =
+    props.noOptionsMessage ||
+    (() => intl.formatMessage(messages.noOptionsMessage));
+  const loadingMessage =
+    props.loadingMessage ||
+    (() => intl.formatMessage(messages.loadingOptionsMessage));
+
+  return (
+    <AsyncSelectInput
+      {...props}
+      {...filterAriaAttributes(props)}
```

Comment:
No point doing this if we're already spreading all props right above this line 😄 
They'll get filtered by the AsyncSelectInput, so it's fine.

Reply:
I was about to ask the same but I forgot to comment on in